stream_alert_cli/test.py

```diff
@@ -271,8 +275,7 @@ def report_output_summary(self):
                 print '\t({}/{}) [{}] {}{}'.format(index + 1, warning_count, failure[0],
                                                    failure[1], color)
 
-    @staticmethod
-    def test_rule(rule_name, test_record, formatted_record):
+    def test_rule(self, rule_name, test_record, formatted_record):
         """Feed formatted records into StreamAlert and check for alerts
         Args:
```

Comment:
`formatted_record`'s description is confusing and has poor grammar. What determines if something is properly formmated? When it says 'properly formatted version of record', what record are we talking about? `test_record`? Please update the description